infra/auth-server/README.md

```diff
@@ -0,0 +1,11 @@
+# `@origin/auth-server`
+
+> TODO: description
```

Comment:
😁

Reply:
Oh, I missed the Readme files. Sorry, will add it 😅